x/bonds/client/rest/query.go

```diff
@@ -141,6 +142,22 @@ func queryCurrentReserveHandler(clientCtx client.Context) http.HandlerFunc {
 	}
 }
 
+func queryAvailableReserveHandler(clientCtx client.Context) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		vars := mux.Vars(r)
+		bondDid := vars[RestBondDid]
+
+		res, _, err := clientCtx.QueryWithData(
+			fmt.Sprintf("custom/%s/%s/%s",
+				types.QuerierRoute, keeper.QueryCurrentReserve, bondDid), nil)
```

Comment:
I think this should be `keeper.QueryAvailableReserve`.

Reply:
Thanks